platform/platform-scheduler/src/test/java/ddf/platform/scheduler/TestCommandJob.java

```diff
@@ -148,12 +153,35 @@ public void testSimpleCommand() throws Exception {
 
     }
 
+    private CommandJob getCommandJob() {
+        CommandJob job = new CommandJob() {
+            @Override
+            public SecurityUtils getDdfSubjectUtils() {
+                SecurityUtils utils = mock(SecurityUtils.class);
+                org.apache.shiro.mgt.SecurityManager sm = mock(
+                        org.apache.shiro.mgt.SecurityManager.class);
+                Subject subject = new SubjectImpl(null, false, "localhost", null, false, sm) {
```

Comment:
Why can't you mock the subject as well?  I think you can use an Answer to do the same thing.

http://site.mockito.org/mockito/docs/current/org/mockito/Mockito.html#answer_stubs